internal/k8s/runtime.go

```diff
@@ -0,0 +1,172 @@
+package k8s
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sRuntime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	"github.com/Shopify/kubeaudit/k8stypes"
+)
+
+// NewTrue returns a pointer to a boolean variable set to true
+func NewTrue() *bool {
+	b := true
+	return &b
+}
+
+// NewFalse returns a pointer to a boolean variable set to false
+func NewFalse() *bool {
+	return new(bool)
```

Comment:
This code existed already so I'm not sure what the original intent was behind the different implementation. I think using `new` is preferred since it initializes an object and returns a pointer to it, which is exactly what we want. But in the case of `true` I don't think there is a way to do it without assigning to a separate variable first.

These functions are used by a bunch of the other auditors to modify a container's security context because all the fields expect pointers
https://godoc.org/k8s.io/api/core/v1#SecurityContext

Reply:
TIL. Thanks for the explanation!